### PR TITLE
docs: Add how to use cloud-localds to boot qemu

### PR DESCRIPTION
**Pull request by powersj**
There is a complete lack of documentation on using cloud-localds with
cloud-init to boot an image locally.

Drive by, added some more whitepapers, blogs, and videos

## Proposed Commit Message
docs: Add how to use cloud-localds to boot qemu

There is a complete lack of documentation on using cloud-localds with
cloud-init to boot an image locally.

## Additional Context
This has come up a lot with the deprecation and removal of the mini.iso. Instead showing people how to use cloud-images properly was lacking the step about customization via cloud-localds.

## Test Steps
Built the docs using `make doc`

## Checklist:

 - [x] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/topics/hacking.html)
 - [x] I have updated or added any unit tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

**Comment by powersj** [comment]
@vorlonofportland could you give this a look over? Would you like more details on cloud-localds or is this enough?

**Review by TheRealFalcon — APPROVED** [review]
Nit: Might want to include the location of the command

**Comment by dermotbradley** [comment]
@powersj FYI I use the following command options:

```
cloud-localds \
  --network-config=network-config-v2.yaml \
  seed.img \
  user-data.yaml \
  meta-data.yaml
```

along with the 3 specified files meta-data.yaml, network-config-v2.yaml, and user-data.yaml. Obviously the "--network-config" option value is changed if you're using a v1 format file.

**Comment by smoser** [comment]
https://asciinema.org/~smoser

there some useful things there if you want to take a look.

they point to http://bit.ly/ci-ubuntu-netconfig

maybe its useful.

**Comment by powersj** [comment]
I have added a comment about where the command comes from.

I've also added another example, links, and paragraph showing network configuration and metadata.

**Comment by smoser** [comment]
I don't want to downplay the *hot*ness factor here, but I might suggest changing the commit message:

```diff
- docs: Add hot to use cloud-localds to boot qemu
+ docs: Add how to use cloud-localds to boot qemu
```

**Comment by powersj** [comment]
Fix the GitHub title and descriptions.

4jb81r

File: doc/rtd/topics/faq.rst [hunk 1]
```diff
@@ -226,12 +226,63 @@ custom network config.
 .. _Instance Configuration: https://linuxcontainers.org/lxd/docs/master/instances
 .. _Custom Network Configuration: https://linuxcontainers.org/lxd/docs/master/cloud-init
 
+cloud-localds
+-------------
+
+The `cloud-localds` command from the cloud-utils generates a disk with user
```
Comment (igalic) [inline doc/rtd/topics/faq.rst:232]: a link to the tool or the utils might be a good idea

**Review by igalic — CHANGES_REQUESTED** [review]
🙋🏻‍♀️

File: doc/rtd/topics/faq.rst (hunk 1 above)
Comment (powersj) [reply doc/rtd/topics/faq.rst:232]: How about to the upstream project like: https://github.com/canonical/cloud-utils/

Comment (powersj) [reply doc/rtd/topics/faq.rst]: Added a link!

**Review by mitechie — APPROVED** [review]

**Comment by igalic** [comment]
unrelatedly, following this tutorial, my cloud-init on FreeBSD ds-identifies OpenStack on my kvm / quemu / libvirt setup

**Pull request merged**